src/drivers/distance_sensor/lightware_sf45_serial/module.yaml

```diff
@@ -41,11 +41,11 @@ parameters:
                       The SF45 mounted facing upward or downward on the frame
               type: enum
               values:
-                0: Rotation upward
-                1: Rotation downward
+                24: Rotation upward
```

Comment:
It won't be used for collision prevention when mounted facing up/down right? Worth to drop a comment in here I feel. As the main usecase of these sensors is collision prevention I assume.

Reply:
The parameter is there if the sensor is mounted upside down, still in the xy plane of the drone.
its used to change the yaw of the measurement, because then a measurement at +100 degs, would be at -100 degs